src/nbl/video/vulkan/profiles/NablaCore.json

```diff
@@ -1,2017 +1,2216 @@
-                }
-            ],
-            "capabilities": [
-                "baseline"
-            ]
-        }
-    }
+{
+    "$schema": "https://schema.khronos.org/vulkan/profiles-0.8-latest.json#",
+    "capabilities": {
+        "VP_LUNARG_generated_profile_block": {
+            "extensions": {
+                "VK_EXT_4444_formats": 1,
+                "VK_EXT_calibrated_timestamps": 2,
+                "VK_EXT_descriptor_indexing": 2,
+                "VK_EXT_extended_dynamic_state": 1,
+                "VK_EXT_extended_dynamic_state2": 1,
+                "VK_EXT_extended_dynamic_state3": 2,
+                "VK_EXT_host_query_reset": 1,
+                "VK_EXT_image_robustness": 1,
+                "VK_EXT_inline_uniform_block": 1,
+                "VK_EXT_pipeline_creation_cache_control": 3,
+                "VK_EXT_pipeline_creation_feedback": 1,
+                "VK_EXT_private_data": 1,
+                "VK_EXT_scalar_block_layout": 1,
+                "VK_EXT_separate_stencil_usage": 1,
+                "VK_EXT_shader_demote_to_helper_invocation": 1,
+                "VK_EXT_shader_subgroup_ballot": 1,
+                "VK_EXT_shader_subgroup_vote": 1,
+                "VK_EXT_subgroup_size_control": 2,
+                "VK_EXT_texel_buffer_alignment": 1,
+                "VK_KHR_16bit_storage": 1,
+                "VK_KHR_8bit_storage": 1,
+                "VK_KHR_bind_memory2": 1,
+                "VK_KHR_buffer_device_address": 1,
+                "VK_KHR_copy_commands2": 1,
+                "VK_KHR_create_renderpass2": 1,
+                "VK_KHR_dedicated_allocation": 3,
+                "VK_KHR_depth_stencil_resolve": 1,
+                "VK_KHR_descriptor_update_template": 1,
+                "VK_KHR_device_group": 4,
+                "VK_KHR_driver_properties": 1,
+                "VK_KHR_dynamic_rendering": 1,
+                "VK_KHR_external_fence": 1,
+                "VK_KHR_external_memory": 1,
+                "VK_KHR_external_semaphore": 1,
+                "VK_KHR_format_feature_flags2": 2,
+                "VK_KHR_get_memory_requirements2": 1,
+                "VK_KHR_image_format_list": 1,
+                "VK_KHR_imageless_framebuffer": 1,
+                "VK_KHR_maintenance1": 2,
+                "VK_KHR_maintenance2": 1,
+                "VK_KHR_maintenance3": 1,
+                "VK_KHR_map_memory2": 1,
+                "VK_KHR_multiview": 1,
+                "VK_KHR_push_descriptor": 2,
+                "VK_KHR_relaxed_block_layout": 1,
+                "VK_KHR_sampler_mirror_clamp_to_edge": 3,
+                "VK_KHR_sampler_ycbcr_conversion": 14,
+                "VK_KHR_separate_depth_stencil_layouts": 1,
+                "VK_KHR_shader_draw_parameters": 1,
+                "VK_KHR_shader_float16_int8": 1,
+                "VK_KHR_shader_float_controls": 4,
+                "VK_KHR_shader_integer_dot_product": 1,
+                "VK_KHR_shader_non_semantic_info": 1,

```

Comment:
nice we can finally require it and not expect drivers to blow up if fed SPIR-V with line-debug info